doc/release/bugs.rst

```diff
@@ -29,13 +29,25 @@ Please include the following information in your email:
 
 A list of known bugs and unimplemented features can be found in
 ``$CHPL_HOME/STATUS``.  Even if you run into a known issue, we
-encourage you to email us a bug report, particularly if it is impeding
+encourage you to send us a bug report, particularly if it is impeding
 your progress.  Depending on the issue, we may be able to suggest a
-workaround or prioritize addressing the bug.
+workaround or to prioritize the development of a fix.
+
+
+Filing Futures
+--------------
+
+If you are a Chapel contributor and familiar with the `Chapel Testing
+System`_, you can file bugs by submitting `Future tests`_.
+
+.. _Chapel testing system: https://github.com/chapel-lang/chapel/blob/master/doc/developer/bestPractices/TestSystem.rst
+.. _Future tests: https://github.com/chapel-lang/chapel/blob/master/doc/developer/bestPractices/TestSystem.rst#futures-a-mechanism-for-tracking-bugs-feature-requests-etc
+
 
 JIRA Issues
 -----------
 
-The Chapel team is hard at work bringing the `JIRA issues site`_ up to date. Bugs can be viewed without a free account.
+The Chapel team is working to bring our `JIRA issues site`_ up to
+date. Bugs can be viewed without a free account.
```

Comment:
What is "without a free account" supposed to mean?  (I didn't write it, just reformatted it to 80 columns).  Are people scared of getting free accounts?  Should it read "with a free account"?  Or "without an account"?